lib/rucio/common/utils.py

```diff
@@ -1435,3 +1435,36 @@ def __call__(self, mtries=3, logger=None):
                     logger.debug(str(e))
                 attempt -= 1
         return self.func(*self.args, **self.kwargs)
+
+
+def tabulate(table, tablefmt='plain', headers=None):
```

Comment:
While this works, I am really wondering how much we gain by doing it. I just cannot fathom that including `tabulate` as a pypi dependency would be an issue for anyone. I could somewhat understand not using any dependencies at all, since there is some merit in that (Although very unrealistic to achieve), but since we are using some anyway, I don't see how these super simple ones are causing any issues. I am not advocating to change it, just noting it.

Reply:
I agree it would probably be unrealistic to aim for zero dependencies. I am happy to leave the tabulate dependency if people think it's unlikely to be a problem in practice.